spec/integration/associations_spec.rb

```diff
@@ -524,18 +524,19 @@ class Page < ActiveFedora::Base
       end
 
       describe "removing association" do
-        subject { LibraryBook.create }
-        before do
-          @p1 = Page.create
-          @p2 = Page.create
-          subject.pages << @p1 << @p2
-          subject.save!
+        let(:p1) { Page.create }
+        let(:p2) { Page.create }
+        let(:book) do
```

Comment:
FYI part of the idea of the `NamedSubject` cop was that you would change `subject { LibraryBook.create }` to `subject(:book) { LibraryBook.create }`. This is fine too but `subject(:name)` might be nice